src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,38 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/product")
 public class ProductController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getProducts")
+    public List<String> getProducts() {
+        return new ArrayList<>();
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "getProduct")
+    public String getProduct() {
+        return "Some product";
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "createProducts")
+    public void createProduct(String newProduct) {
```

Comment:
Drobna sprawa, ale tutaj dobrze by było zachować jednolity kod poleceń. Value masz "createProducts", a metodę createProduct - może się to później mylić przy wpisywaniu poleceń.

Reply:
Słusznie, poprawiłem.